### PR TITLE
implement pop() and cull() on multiqueue/slabseqn

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/lib/stormtypes.py [hunk 1]
```diff
@@ -2006,19 +2006,22 @@ async def _methQueueGet(self, offs=0, cull=True, wait=True):
...
-        valu = await self.runt.dyncall('cortex', todo, gatekeys=gatekeys)
-        if valu is not None:
-            await self._methQueueCull(valu[0])
+        # emulate the old behavior on no argument
+        if offs is None:
+            todo = s_common.todo('coreQueueGets', self.name, 0, cull=True, wait=False)
+            async for item in self.runt.dyniter('cortex', todo, gatekeys=gatekeys):
+                await self._methQueueCull(item[0])
+                return item
+            return
```
Comment (vEpiphyte) [inline synapse/lib/stormtypes.py:2020]: calling pop without an offset should cover this

File: synapse/lib/stormtypes.py [hunk 2]
```diff
@@ -2006,19 +2006,22 @@ async def _methQueueGet(self, offs=0, cull=True, wait=True):
...
-        todo = s_common.todo('coreQueueGet', self.name, offs, cull=cull, wait=wait)
+        offs = await toint(offs, noneok=True)
         gatekeys = self._getGateKeys('get')
 
-        valu = await self.runt.dyncall('cortex', todo, gatekeys=gatekeys)
-        if valu is not None:
-            await self._methQueueCull(valu[0])
+        # emulate the old behavior on no argument
+        if offs is None:
+            todo = s_common.todo('coreQueueGets', self.name, 0, cull=True, wait=False)
```
Comment (vEpiphyte) [inline synapse/lib/stormtypes.py:2016]: Using ``coreQueueGet`` avoids the need for the iterator

File: synapse/lib/stormtypes.py (hunk 1 above)
Comment (invisig0th) [reply synapse/lib/stormtypes.py:2020]: no.  cull and pop are meant to be very different semantics.

File: synapse/lib/stormtypes.py (hunk 2 above)
Comment (invisig0th) [reply synapse/lib/stormtypes.py:2016]: But doesn't implement the backward compatible behavior.

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**